index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Mobile First Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+    <!--MAIN CONTENT-->
+    <main class="main">
+
+        <!--Header-->
+        <header>
+            <h1>John Doe</h1>
+
+            <div id="navig">
+                <label for="humberger" id="hum">&#9776;</label>
+                <input type="checkbox" id="humberger"/>
+                <div id="navitems">
+                    <a href="#">Portfolio</a>
+                    <a href="#">About</a>
+                    <a href="#">Contact</a>
+                    <a href="#"><img alt="Mail" src="/assets/mail-icon.svg" width="20" height="30"></a>
+                </div>
+                </div>
+</header>
+<!--Toolbar-->
+<section id="toolbar">
+    <div id="heytext">
+        <p>Hey There.</p>
+        <p>I’m Alex</p>
+    </div>
```

Comment:
- [ ] This is the main heading of your page and should be written in a proper `<h>` tag. Please take a look at [this article](https://www.reliablesoft.net/h1-tag/) to understand more about it.
 - [ ] Please follow the Figma template as much as possible. You should use the same font-family, font-size, font-weight, etc. Try inspecting in the element in Figma, it should display in the right side the CSS elements used.
 -Your design:
<img width="410" alt="image" src="https://user-images.githubusercontent.com/61919617/186482415-92e55171-5804-410a-9e38-b263671f8407.png">
-Figma design:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/61919617/186482554-a8d9e0ec-4c5b-4269-905b-359aa8e9ecbd.png">